soh/src/code/z_player_lib.c

```diff
@@ -572,6 +573,23 @@ uint8_t Player_IsCustomLinkModel() {
            (LINK_IS_CHILD && ResourceGetIsCustomByName(gLinkChildSkel));
 }
 
+// Alternate Equipment Loading function. (global function)
+// Checks if player can use new loading method, for Bottles Only.
+uint8_t Player_CanUseNewLoadingMethodBottle(Player* this) {
+    if (!CVarGetInteger("gAltLinkEquip", 1)) {
```

Comment:
Should default to off.

```suggestion
    if (!CVarGetInteger("gAltLinkEquip", 0)) {
```